app/src/main/res/values/strings.xml

```diff
@@ -302,102 +302,103 @@
     <string name="menu">Menu</string>
     <string name="open_in_pseudo_account">Open in emulated account on %1$s</string>
 
-    <string name="color_and_background">Color and Background…</string>
-    <string name="column_background">Column\'s background</string>
+    <string name="color_and_background">Colour and Background…</string>
+    <string name="column_background">Column background</string>
     <string name="pick_image">Pick an image</string>
     <string name="pick_images">Pick image(s)…</string>
 
     <string name="image_alpha">Image alpha</string>
     <string name="image">Image</string>
     <string name="column_header">Column header</string>
-    <string name="foreground_color">Foreground color</string>
-    <string name="hashtag_and_visibility_not_match">this message contains hashtags, but message visibility is not public. normally hashtags are searchable only in public messages. Are you sure?</string>
+    <string name="foreground_color">Foreground colour</string>
+    <string name="hashtag_and_visibility_not_match">Send non-public toot containing hashtags, (which are not normally searchable in public messages)?</string>
```

Comment:
This sentence will be a problem when translating to other languages.

Reply:
How so?